packages/network/src/workers/SyncWorker.spec.ts

```diff
@@ -154,6 +154,7 @@ describe("Sync.worker", () => {
   it("should pass live events to the output", async () => {
     input$.next({
       checkpointServiceUrl: "",
+      streamServiceUrl: "",
```

Comment:
Let's add one test for "should sync from stream service if stream service url is provided"